deploy-clojars/action.yml

```diff
@@ -0,0 +1,89 @@
+name: Clojars Deployment
+
+description: Build JAR and Deploy to Clojars
+
+inputs:
+  group-id:
+    description: The Clojars Group ID. Defaults to 'com.yetanalytics'.
+    required: false
+    default: 'com.yetanalytics'
+
+  artifact-id:
+    description: The Clojars Artifact ID (i.e. the name of the lib itself)
+    required: true
+
+  version:
+    description: >
+      The version string. Defaults to the Git branch or tag associated with
+      the event that triggered this action. NOTE: If the tag is prefixed
+      (e.g. 'v1.0.0'), you should remove the prefix, then pass in the
+      truncated tag as the arg (e.g. pass in `${GITHUB_REF#refs\/tags\/v}`).
+    required: false
+    default: ${{ github.ref_name }}
```

Comment:
This is quite clever, but also I'd be a little concerned about accidentally pushing some weird versions to clojars. Maybe just make this a dumb string and let the caller worry about it?

Reply:
So something like `default: 'LATEST'`?

Reply:
I don't know if clojars lets you push something like that. I think the version is sufficiently important to make it required and NOT provide a default

Reply:
Or put another way, if you don't provide the arg why are you using the action in the first place?